assets/css/directives.css

```diff
@@ -0,0 +1,9 @@
+.v-loading {
+  @apply relative opacity-60 pointer-events-none select-none overflow-hidden min-h-[30px] min-w-[30px];
+}
+
+.v-loading::after {
+  @apply content-[''] absolute z-20 w-4 h-4 border-2 rounded-full animate-[spin_600ms_linear_infinite] border-black border-t-transparent;
+  top: calc(50% - 8px);
+  left: calc(50% - 8px);
```

Comment:
Tried to add everything in tailwind, but this part is something I couldn't find a solution for

Reply:
Actually did it 👌